site/en/install/pip.md

```diff
@@ -211,7 +229,31 @@ The following NVIDIA® software are only required for GPU support.
     pip install tensorflow
     ```
 
-    ### 4. Verify the installation
+    **Note:** Do not install TensorFlow with `conda`. It may not have the latest stable version. `pip` is recommended since TensorFlow is only officially released to PyPI.
+
+    ### 5. Virtual environment configuration
+
+    You can skip this section if you only run TensorFlow on the CPU.
+    
+    Note: Symbolic links are only necessary in case the intended way doesn't work, i.e. the components aren't being
```

Comment:
```suggestion
    Check if TensorFlow successfully connected to the GPU
    
    ```
    import tensorflow as tf
    print("GPUs:", tf.config.list_physical_devices('GPU'))
    print(tf.constant(1) + tf.constant(1)) 
    ```

    If that doesn't work, you may need to add symbolic links because components aren't being
```